taxonomy/app/database.py

```diff
@@ -17,6 +18,16 @@
 
 engine = create_engine(SQLALCHEMY_DATABASE_URL)
 
+# Ensure LTREE extensions is installed
+with engine.connect() as conn:
+    try:
+        conn.execute(sqlalchemy.sql.text("CREATE EXTENSION IF NOT EXISTS ltree"))
+    except sqlalchemy.exc.ProgrammingError as err_:
+        # Exctension installed, just skip error
+        if "DuplicateObject" not in str(err_):
+            raise err_
```

Comment:
May we move it to some sort of function? and add it as pre-setup step for app?

Reply:
For instance, if it's not done already, create setup_db function to adjust all db settings before starting the app

Reply:
I've decided to move it directly to alembic migrations as long as it set our database up and it seems appropriate having this logic there